fuse_constraints/include/fuse_constraints/normal_prior_pose_2d_cost_functor.h

```diff
@@ -66,6 +67,8 @@ namespace fuse_constraints
 class NormalPriorPose2DCostFunctor
 {
 public:
+  FUSE_MAKE_ALIGNED_OPERATOR_NEW();
```

Comment:
As before, for the `normal_delta` version of `pose_2d`, since https://github.com/locusrobotics/fuse/pull/65/files#diff-e91722dbdc7432f8dac96adfee2b8dbbR92 is dynamic, you don't need this.